src/AI/Muriel.java

```diff
@@ -292,17 +292,12 @@ public GameAction getNextAction(GameMap gameMap)
         shouldResupply = true;
       }
       // If we are out of ammo.
-      if( unit.weapons != null && unit.weapons.size() > 0 )
+      if( unit.ammo == 0 )
```

Comment:
Won't this be true for APCs? You have to also check that the unit has weapons before worrying about reloading.

Reply:
IIRC, they're initialized to -1.
Either way, it's immaterial at the moment because Muriel doesn't build them.

Reply:
Sigh. True enough.